juno/assetlist.json

```diff
@@ -1242,6 +1242,27 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/spacer.png"
       }            
     },
+    {
+      "description":"Light: LumenX community DAO treasury token",
+      "type_asset": "cw20",
+          "denom_units":[
+             {
+                "denom":"nlight",
```

Comment:
you're free to add 'nlight' as an alias of the base, but the base denom will be something like 'cw20:juno1a2b3....'